content/source/docs/enterprise-beta/api/policy-checks.html.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: enterprise2
+page_title: "Policy Checks - API Docs - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-api-policy-checks"
+---
+
+# Policy Checks API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+## List policy checks
+
+This endpoint lists the policy checks in a run
+
+| Method | Path           |
+| :----- | :------------- |
+| GET | /runs/:run_id/policy-checks |
```

Comment:
Any reason we leave out `/api/v2` in these? Might be confusing unless it is implied elsewhere.

Reply:
The examples all show that the subpath is `/api/v2`. However, to make this more clear I will add a section to the `api/index.html.md` page explaining this.